models/users.js

```diff
@@ -1760,6 +1760,38 @@ if (Meteor.isServer) {
       });
     }
   });
+
+  /**
+   * @operation create_user_token
+   *
+   * @summary Create a user token
+   *
+   * @description Only the admin user (the first user) can call the REST API.
+   *
+   * @param {string} userId the ID of the user to delete
```

Comment:
Isn't this about creating token? Not deleting user?

Reply:
oups sorry